lib/git-interface/comment.ts

```diff
@@ -0,0 +1,55 @@
+export class GitComment
+{
+    public owner: string;
+    public repo: string;
+    public issue_id: number;
+    public id: string;
+    public body: string;
```

Comment:
we should be able to get both the rawtext and the html versions of the comment via githubs api, do we have access to those here?

Reply:
Not currently. I'm looking into how to do that.

Reply:
Various URLs (both API and HTML) are now stored in each of the classes and interfaces.